File: JY/8/8.25/카펫.py
def solution(brown, yellow):
    answer = []
    for i in range(1, int(yellow**1/2)+2):

        if yellow%i==0:
            print(i,999)
            if brown==(((yellow//i)+2+ i+2)*2-4):
                return [yellow//i+2, i+2]

File: YJ/20200904/BruteForce/카펫.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(int brown, int yellow) {
    vector<int> answer;
    int cnt = (brown + 4) / 2;
    int height = 3, width = cnt - 3;
    
    while (true) {
        if ((height - 2) * (width - 2) == yellow) {
            answer.push_back(width);
            answer.push_back(height);
            break;
        } else {
            height ++;
            width --;
        }
    }
    
    return answer;
}
